gatsby/src/components/DetailsPageLayout/DetailsPageLayout.module.scss

```diff
@@ -5,3 +5,68 @@
     margin-top: 0;
   }
 }
+
+.toggleButton {
+  appearance: none;
+  background: core.$nds-colour-nice-grey-3;
+  border: 1px solid core.$nds-colour-border;
+  color: core.$nds-colour-link;
+  cursor: pointer;
+  display: block;
+  line-height: 1;
+  margin: 0;
+  /* stylelint-disable indentation, function-parentheses-space-inside */
+  padding: core.rem(
+    core.$nds-spacing-medium core.$nds-spacing-large core.$nds-spacing-medium
+      core.$nds-spacing-small
+  );
+  position: relative;
+  text-align: left;
+  text-decoration: none;
+  width: 100%;
+
+  &:visited {
+    color: core.$nds-colour-link;
+  }
+
+  &:focus {
+    @include core.nds-default-focus-style;
+    z-index: 1;
+  }
+
+  /* stylelint-disable selector-class-pattern */
+  + :global(.stacked-nav) {
+    display: none;
+
+    // Non-JS fallback
+    &:target {
+      display: block;
+    }
+  }
+
+  &[aria-expanded='false'] + :global(.stacked-nav) {
+    display: none;
+  }
+
+  &[aria-expanded='true'] {
+    border-bottom: 0;
+
+    .icon {
+      transform: translateY(-50%) rotate(180deg);
+    }
+
+    + :global(.stacked-nav) {
+      display: block;
+    }
+  }
+
+  @include core.mq($from: sm) {
+    display: none;
+  }
+}
+
+@include core.mq($from: sm) {
+  :global(.stacked-nav) {
+    display: block !important;
```

Comment:
Is there an alternative to using `!important` here - could stack up specificity instead perhaps? Something along the lines of `:global(.stacked-nav.stacked-nav)` might work and would give you the ability to apply further overrides, which `!important` would not.